taipy/gui_core/_adapters.py

```diff
@@ -206,11 +206,17 @@ def __get_data(self, dn: DataNode):
                 )
                 if isinstance(value, float) and math.isnan(value):
                     value = None
+                error = None
+                if val_type not in ("date", "int", "float", "string"):
+                    try:
+                        json.dumps(value)
```

Comment:
do you want to use the JsonAdapter that we use in the gui

Reply:
No I don't :-)
I tried it but it throws a warning instead of an Exception 
I would have to catch exceptions and warnings
And I'm not sure we need the extra support of JsonAdapter for datanode values that are not supported (ie data identified as tabular will not go through this)

Reply:
but thanks for reading thoroughly :-)